cmake/modules/CUDA.cmake

```diff
@@ -10,9 +10,16 @@ endif()
 include(CheckCXXCompilerFlag)
 check_cxx_compiler_flag("-std=c++14"   SUPPORT_CXX14)
 
-set(dgl_known_gpu_archs "35 50 60 70")
+set(dgl_known_gpu_archs "35" "50" "60" "70")
```

Comment:
What is the reason to change "35 50 60 70" to "35 50 60 70"?

Reply:
To make it a list and then in 
https://github.com/dmlc/dgl/blob/3e8af7d66d5dce631f1bd6c5e1176022c15b5f7a/cmake/modules/CUDA.cmake#L157-L159
we can easily get the last item in `__cuda_arch_bin` as `__cuda_arch_ptx`.